semantic_release/defaults.cfg

```diff
@@ -6,6 +6,7 @@ check_build_status=false
 hvcs=github
 commit_parser=semantic_release.history.angular_parser
 upload_to_pypi=true
+upload_to_release=true
```

Comment:
Why should this be default true?

Reply:
Since it's uploading the same files that are being uploaded to PyPI, I just selected the same default as that. I'm happy to change it if you like.

Reply:
Let's leave it :)